api/renter.go

```diff
@@ -35,9 +35,18 @@ type (
 		FinancialMetrics modules.RenterFinancialMetrics `json:"financialmetrics"`
 	}
 
+	// RenterContract represents a contract formed by the renter.
+	RenterContract struct {
+		FileContract types.FileContract         `json:"filecontract"`
+		ID           types.FileContractID       `json:"id"`
+		LastRevision types.FileContractRevision `json:"lastrevision"`
+		NetAddress   modules.NetAddress         `json:"netaddress"`
+		Size         uint64                     `json:"size"`
```

Comment:
open to feedback on which fields should be included. Specifically, I opted to precalculate `Size` instead of returning the number of Merkle roots. I think this is what 99% of API consumers care about.